src/Client.svelte

```diff
@@ -1,16 +1,2 @@
-<script lang="ts">
-  import { createClient, setClient } from "@urql/svelte";
-
-  // import { selectedNetwork } from "./stores";
-
-  import { AddressBook } from "rain-sdk";
-
-  let client = createClient({
-    // url: $selectedNetwork.graphUrl,
-    url: AddressBook.getSubgraphEndpoint(80001),
-  });
-
-  setClient(client);
-</script>
 
 <slot />
```

Comment:
@rouzwelt is this component redundant now? should we just delete it?

Reply:
ya sure. Just tested by commented this component, working fine

Reply:
you're right, and it is been resolved